client/my-sites/posts/post-controls.jsx

```diff
@@ -144,14 +145,14 @@ const getAvailableControls = props => {
 
 	if ( controls.more.length ) {
 		controls.main.push( {
-			className: 'more',
+			className: 'more' + ( current === 'more' ? ' is-active' : '' ),
 			icon: 'ellipsis',
 			onClick: onShowMore,
 			text: translate( 'More' ),
 		} );
 
 		controls.more.push( {
-			className: 'back',
+			className: 'back' + ( current === 'back' ? ' is-active' : '' ),
```

Comment:
Could we loop through `controls.more` and `controls.main` . and apply proper class to current item instead of all these 

Something like:
```
controls.more.foreach( control => {
if( control.name === current)
control.classname += 'active'
 }  )
```